src/cljs/athens/coeffects.cljs

```diff
@@ -1,15 +1,21 @@
 (ns athens.coeffects
   (:require
-    [re-frame.core :refer [reg-cofx]]))
+    [re-frame.core :as rf]))
 
 
-(reg-cofx
-  :local-storage
+(rf/reg-cofx
+  :local-storage/get
   (fn [cofx key]
-    (assoc cofx :local-storage (js/localStorage.getItem key))))
+    (assoc cofx :local-storage (or
+                                 (js/localStorage.getItem (str key))
+                                 (js/localStorage.getItem key)))))
 
 
-(reg-cofx
-  :local-storage-map
-  (fn [cofx {:keys [ls-key key]}]
-    (assoc cofx key (js/localStorage.getItem ls-key))))
+(rf/reg-fx
+  :local-storage/set
```

Comment:
Jeff `:local-storage/set!` is already there in the codebase, would it not be better to modify that? 
Some other coeffect like `:local-storage-map` is still needed for cases when we need multiple values from the local storage.

Reply:
Hey Sid, was planning to just remove `:local-storage/set!`. 

I don't think `:local-storage-map` accesses multiple values from local-storage

Reply:
Yeah `:local-storage-map` does not acces multiple values from local-storage but it is general in the sense we can mention both the value we want to access from localstorage and the key that we want to assign to this accesed value, for e.g:

```
(reg-event-fx
    :local-storage/get-db-filepath
    [(inject-cofx :local-storage "db/filepath")
     (inject-cofx :local-storage-map {:ls-key "db/remote-graph-conf"
                                      :key    :remote-graph-conf})]
    (fn [{:keys [local-storage remote-graph-conf]} _]
```

Here we are getting 2 values from the localStorage, using `local-storage/get` we can only access one value.